app/templates/administrative-units/administrative-unit/ministers/new.hbs

```diff
@@ -82,9 +82,9 @@
                   of probeer een andere zoekopdracht. Indien een persoon
                   ontbreekt, meldt dit aan
                   <a
-                    href="mailto:organisaties.abb@vlaanderen.be"
+                    href="mailto:{{config "contactEmail"}}"
                     class="au-c-link"
-                  >organisaties.abb@vlaanderen.be</a>.
+                  >{{config "contactEmail"}}e</a>.
```

Comment:
There's a ninja `e` in there :smile:.

Reply:
Or maybe a mosquito on your monitor :smile_cat: